src/v/cluster/members_manager.h

```diff
@@ -176,8 +178,10 @@ class members_manager {
     // guarantee that the UUID has already been registered before calling.
     model::node_id get_node_id(const model::node_uuid&);
 
+    // Initialize `_id_by_uuid` in the beginning
```

Comment:
nit: "in the beginning" is a bit vague. How about:

"Initializes `_id_by_uuid`; should be called once only when bootstrapping a cluster."